tasks/gulp-tests.js

```diff
@@ -404,100 +322,4 @@ gulp.task('test:api-v3', (done) => {
     'test:api-v3:integration',
     done
   );
-});
-
-// Old tests tasks
-/*
-gulp.task('test:api-v3', ['test:api-v3:unit', 'test:api-v3:integration']);
-
-gulp.task('test:api-v3:watch', ['test:api-v3:unit:watch', 'test:api-v3:integration:watch']);
-
-gulp.task('test:api-v3:unit', (done) => {*/
-//  runMochaTests('./test/api/v3/unit/**/*.js', null, done)
-/*});
-
-gulp.task('test:api-v3:unit:watch', () => {
-  gulp.watch(['website/server/**', 'test/api/v3/unit/**'], ['test:api-v3:unit']);
-});
-
-gulp.task('test:api-v3:integration', ['test:prepare:server'], (done) => {
-  process.env.API_VERSION = 'v3';
-  awaitPort(TEST_SERVER_PORT).then(() => {*/
-//    runMochaTests('./test/api/v3/integration/**/*.js', server, done)
-/*  });
-});
-
-gulp.task('test:api-v3:integration:watch', ['test:prepare:server'], () => {
-  process.env.RUN_INTEGRATION_TEST_FOREVER = true;
-  gulp.watch(['website/server/**', 'test/api/v3/integration/**'], ['test:api-v3:integration']);
-});
-
-gulp.task('test:api-v3:safe', ['test:prepare:server'], (done) => {
-  awaitPort(TEST_SERVER_PORT).then(() => {
-    let runner = exec(
-      testBin(API_V3_TEST_COMMAND),
-      (err, stdout, stderr) => {
-        testResults.push({
-          suite: 'API V3 Specs\t',
-          pass: testCount(stdout, /(\d+) passing/),
-          fail: testCount(stdout, /(\d+) failing/),
-          pend: testCount(stdout, /(\d+) pending/)
-        });
-        done();
-      }
-    );
-    pipe(runner);
-  });
-});
-
-gulp.task('test:all', (done) => {
-  runSequence(
-  //'test:e2e:safe',
-  //'test:common:safe',
-  //'test:content:safe',
-  // 'test:server_side:safe',
-  //'test:karma:safe',
-  //'test:api-legacy:safe',
-  //'test:api-v2:safe',
-  'test:api-v3:safe',
-  done);
-});
-
-gulp.task('test', ['test:all'], () => {
-  let totals = [0,0,0];
-
-  console.log('\n\x1b[36m\x1b[4mHabitica Test Summary\x1b[0m\n');
-  testResults.forEach((s) => {
-    totals[0] = totals[0] + s.pass;
-    totals[1] = totals[1] + s.fail;
-    totals[2] = totals[2] + s.pend;
-    console.log(
-      `\x1b[33m\x1b[4m${s.suite}\x1b[0m\t`,
-      `\x1b[32mPassing: ${s.pass},\t`,
-      `\x1b[31mFailed: ${s.fail},\t`,
-      `\x1b[36mPending: ${s.pend}\t`
-    );
-
-    if (s.pass === 0) {
-      console.error('ERROR: Detected a test suite with 0 passing tests. Something may be wrong causing the build to error.');
-      process.exit(1);
-    }
-  });
-
-  console.log(
-    '\n\x1b[33m\x1b[4mTotal:\x1b[0m\t\t\t',
-    `\x1b[32mPassing: ${totals[0]},\t`,
-    `\x1b[31mFailed: ${totals[1]},\t`,
-    `\x1b[36mPending: ${totals[2]}\t`
-  );
-
-  kill(server);
-
-  if (totals[1] > 0) {
-    console.error('ERROR: There are failing tests!');
-    process.exit(1);
-  } else {
-    console.log('\n\x1b[36mThanks for helping keep Habitica clean!\x1b[0m');
-    process.exit();
-  }
-});*/
+});
```

Comment:
I don't think we have a policy on trailing new lines on file endings. I added an editorconfig file that will auto format if you have the plugin for your preferred text editor, but it never really caught on.